2024-12-21 04:44:02 - Running application...
Traceback (most recent call last):
  File "/home/retro/Documents/crudeirc/crudeirc.py", line 1280, in <module>
    app = CrudeIRC(root)
          ^^^^^^^^^^^^^^
  File "/home/retro/Documents/crudeirc/crudeirc.py", line 137, in __init__
    self.server_details = self.active_details["servers"]
                          ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'servers'
Traceback (most recent call last):
  File "/home/retro/Documents/crudeirc/crudeirc.py", line 1280, in <module>
    app = CrudeIRC(root)
          ^^^^^^^^^^^^^^
  File "/home/retro/Documents/crudeirc/crudeirc.py", line 137, in __init__
    self.server_details = self.active_details["servers"]
                          ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'servers'
2024-12-21 04:44:03 - Application finished. Waiting for user input before exiting...
2024-12-21 04:44:05 - User pressed Enter. Exiting...
